android/src/fdroid/play/listings/ar/release-notes.txt

```diff
@@ -1,14 +1,13 @@
-• بيانات OpenStreetMap جديدة اعتبارًا من 10 يوليو 2023
-• Fixed several crashes
-• دعم ألوان Garmin و OsmAnd و altitude في مسارات GPX المستوردة
-• تم إصلاح ETA للمشاة للمنحدرات الكبيرة
-• توجيه أفضل للسيارة
-• تحسين ترتيب البحث
-• تم نقل زر المشاركة إلى صفحة المكان
-• تحديث العديد من الترجمات
-• تمت إضافة المتاجر الجديدة وأماكن الفعاليات
-• تحسين أنفاق وجسور المياه
-• سكك حديدية أكثر تفصيلا
-• الحدود المضافة=المنطقة_المحمية
+• New OpenStreetMap data as of August 14
+• Fresh Wikipedia articles
+• Fixed search for addresses in Czech and Slovakia
+• Added missing isolines for some regions of Argentina, Chile, Kazakhstan, Tibet, USA, Russia
+• More dense isolines in some areas
+• Fixed bug on Nexus 5X
+• Added ferry routes to the World map
+• Redesigned parking icons
+• Revamped priorities system for POIs
```

Comment:
What does it mean?

Reply:
Detailed news without 500 chars limit explains it.